apps/web-giddh/src/app/invoice/preview/invoice.preview.component.ts

```diff
@@ -1860,4 +1876,46 @@ export class InvoicePreviewComponent implements OnInit, OnChanges, OnDestroy {
             this.getVoucher(this.isUniversalDateApplicable);
         }
     }
+
+    /**
+     * Creates the E-invoice in bulk
+     *
+     * @memberof InvoicePreviewComponent
+     */
+    public createBulkEInvoice(): void {
+        const requestObject = {
+            model: {
+                voucherNumbers: this.selectedInvoicesList.map(item => item.voucherNumber),
+                voucherType: this.selectedVoucher
+            },
+            actionType: 'einvoice'
+        };
+        this.store.dispatch(this.invoiceActions.generateBulkEInvoice(requestObject));
+    }
+
+    /**
+     * Returns the E-invoice tooltip text
+     *
+     * @private
+     * @param {ReceiptItem} item Current item
+     * @return {string} E-invoice status
+     * @memberof InvoicePreviewComponent
+     */
+    private getEInvoiceTooltipText(item: ReceiptItem): string {
```

Comment:
Multi-lingual is not supported on STAGE and higher env. thererfore have to use label values directly